src/Repository/ProjectRepository.php

```diff
@@ -56,4 +56,14 @@ public function addConditionState(QueryBuilder $query, string $state): void
             ->andWhere('p.state = :state')
             ->setParameter('state', $state);
     }
+
+    /**
+     * Filter projects according to a Shuttle.
+     */
+    public function addConditionShuttle(QueryBuilder $query, string $shuttle): void
+    {
+        $query
+            ->andWhere('p.shuttle = :shuttle')
+            ->setParameter('shuttle', $shuttle);
```

Comment:
Then this will change for 
```suggestion
            ->setParameter('shuttle', $shuttle->getId());
```